### ISSUE TITLE
Incorporate logo

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#72]
Waiting on final confirmation of logo and where to include in the designs.

**Comment by vickimcfadden** [comment]
@PowderD @ekidenda could you provide the final, approved logo? cc @klohman

**Comment by klohman** [comment]
Assuming the final and approved logo was the one I sent over, here are some mockups of including the logo in the header on the site. 

### Homepage header 
![Image](https://github.com/GSA/oasis-plus/assets/18366192/490ec3ed-c738-4b27-a90e-167fffca58f8)

### Internal page header 
![Image](https://github.com/GSA/oasis-plus/assets/18366192/c7810a49-8867-4638-830c-265490c339ed)

@ekidenda @PowderD Let me know if this placement works.

**Comment by ekidenda** [comment]
Assuming the logo is approved and final, it looks good to me. @PowderD please weigh in before we move this forward.

On a separate note, please de-hyphenate 'Governmentwide'.

**Comment by PowderD** [comment]
@klohman @vickimcfadden @ekidenda Yes, this  is the final approved logo.  This looks good for the website.  I'm moving it over to completed.

**Comment by klohman** [comment]
@PowderD Those screenshots aren't live yet so don't move this to done just yet. 

@mark-meyer @weiwang-gsa This is ready to be picked up. 

Homepage logo - 
![oasisplus-updated-inverse](https://github.com/GSA/oasis-plus/assets/18366192/7ff9203e-32cf-457f-90de-92faca7df9bc)

Internal page logo - 
![oasisplus-updated](https://github.com/GSA/oasis-plus/assets/18366192/08b5401f-71d2-432b-ab13-f6f275fd8db5)

**Comment by PowderD** [comment]
Sounds good.  I didn't mean to say move it to done :)

*Jon DeWolfe  |  *
Program Manager Buyer Engagement
Services Marketplace
Federal Acquisition Service, U.S. General Services Administration
|(: 208.830.8517 | *: ***@***.***

**Comment by mark-meyer** [comment]
@klohman logos!! https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/mmeyer/oasis-logos/

**Comment by klohman** [comment]
@mark-meyer Looks good - ready for QA!

**Comment by PowderD** [comment]
@klohman @mark-meyer @ekidenda This looks great.  I'm good with moving it over. 

Additionally, we heard back from the trademark attorney this morning and they don't see any issues with getting the logo trademarked.  We're starting the application process once we launch the website.
